x-pack/plugins/security_solution/public/exceptions/components/create_shared_exception_list/index.tsx

```diff
@@ -75,6 +80,15 @@ export const CreateSharedListFlyout = memo(
       setNewListDetails({ ...newListDetails, [detailProperty]: value });
     };
 
+    useEffect(() => {
```

Comment:
Just to understand, Is there any a difference between using `useEffect` and when we add the navigation to the newly created list in the `handleCreateSuccess` 
https://github.com/elastic/kibana/blob/c814dcf85b8a722353fb765e3cd04a861bf73495/x-pack/plugins/security_solution/public/exceptions/components/create_shared_exception_list/index.tsx#L117?